### PR TITLE
Sending Changes discussed via Slack

### PR DESCRIPTION
**Pull request by gshields5541**
1. Changed CSS coding to restyle landing.php, registration.php, search_events.php, layout.php and the signing modal
2. Corrected an issue with the carousel in landing.php (I hope)
3. Added a banner logo (FunLogoBanner.png).
4. Replaced the pics that was part of the carousel slideshow with pics (1200 px by 600 px)
5. Moved the Search Events link that was in layout.php to header.php and created a nav link to the search_events.php page.
6. Changed the Fun_Logo1.png file to a more "serious" style
7. Implemented a color scheme discussed by the Front-End team via Slack using the following colors: #316117
#90B644
#FFFFFF
#E98341
#F2FEFE
#2D6077

### CODE REVIEW

**Comment by mviencek** [comment]
gerald please pull the latest from development and resolve your conflicts.

**Comment by mviencek** [comment]
youre missing the latest code

**Comment by mviencek** [comment]
cssbranch
The styling is also off on a few of the pages

**Comment by gshields5541** [comment]
You mean origin/Development?

**Comment by mviencek** [comment]
yes Development, not development.  Mekone named his development

**Comment by gshields5541** [comment]
Just double-checking:

Two files: I got modified version Yours say they are deleted. I’m accepting your changes.

Gerald

File: models/event.php [hunk 1]
```diff
@@ -4,9 +4,10 @@ class event
 
     private $eventId;
     private $organizationId;
+    private $creatorId;
     private $name;
     private $description;
-    private $address;
+    private $location;
```
Comment (mcontado) [inline models/event.php]: why are these files being changed? we are not supposed to change the files under models directory. please do not change this file.

Comment (gshields5541) [reply models/event.php]: I didn't changed that file

Comment (mcontado) [reply models/event.php]: yeah. but just curious why it was being added here

**Review by mcontado — CHANGES_REQUESTED** [review]
- [ ] Search Events are off, not showing search bar and maps.
- [ ] Do not include models/event.php changes.

**Comment by gshields5541** [comment]
Clint told me to pull the latest from Development, thus I’m assuming the change was made from that pull. But I didn’t change that file.

**Comment by gshields5541** [comment]
I’ve only worked on CSS changes associated with the page. The Maps never worked.

Gerald

**Comment by gshields5541** [comment]
Have you checked that entire file? That’s a variable name change and it’s all over the events.php file.

**Comment by mviencek** [comment]
im the one that asked you to pull a current copy of dev.  your event.php file is different from the one in dev.  when you submit a pull request, it shows us the difference between what is in dev and what you are asking to pull

**Comment by mviencek** [comment]
it looks like you merged with mekones at some point and time.  those variables are only called that on his branch.  unfortunately he named his branch development with a lower case d

**Comment by mviencek** [comment]
the one you should be pulling from is Development with an uppercase D

**Comment by gshields5541** [comment]
Okay, and the last pull was from Development with a capital “D"

**Comment by mviencek** [comment]
you must have pulled from development at one point in time though.  now you have a mix of both

**Comment by gshields5541** [comment]
I’ve copied the original events.php file from Development and pasted it over what I’ve have.

**Comment by mviencek** [comment]
cool that should work for that file.  dont forget to commit and push

**Comment by mcontado** [comment]
The maps works like this, with markers coming from API:

screenshot 2018-05-28 21 14 18

**Comment by mcontado** [comment]
pull the latest one by: `git pull origin Development`

**Review by mcontado — APPROVED** [review]

**Pull request merged**